cmd/influx/main_test.go

```diff
@@ -236,3 +254,19 @@ func newNode(config *influxd.Config, path string) (*influxd.Node, error) {
 	}
 	return node, nil
 }
+
+func newPort() int {
+	l, e := net.Listen("tcp", ":0")
+	defer l.Close()
+	if e != nil {
+		panic(e)
+	}
+
+	parts := strings.Split(l.Addr().String(), ":")
+	p, err := strconv.Atoi(parts[len(parts)-1])
+	if err != nil {
+		panic(err)
+	}
+	return p
```

Comment:
Unfortunately we confirmed that on Ubuntu 14.04, port collision occur in this scenario.